src/vision/videoIndexerV2.js

```diff
@@ -0,0 +1,319 @@
+const commonService = require('../commonService');
+const querystring = require('querystring');
+
+/**
+ * Video Indexer is a cloud service that enables you to extract the following insights from your videos using artificial intelligence technologies:
+ */
+class videoIndexerV2 extends commonService {
+    /**
+     * Constructor.
+     * 
+     * @param {Object} obj
+     * @param {string} obj.apiKey
+     */
+    constructor({ apiKey }) {
+        const endpoint = "api.videoindexer.ai";
+        super({ apiKey, endpoint });
+        this.endpoints = [
+            endpoint
+        ];
+        // https://docs.microsoft.com/en-us/azure/media-services/media-services-media-encoder-standard-formats
+        this.supportedFormats = ['flv', 'mxf', 'gxf', 'ts', 'ps', '3gp', '3gpp', 'mpg', 'wmv', 'asf', 'avi', 'mp4', 'm4a', 'm4v', 'isma', 'ismv', 'dvr-ms', 'mkv', 'wav', 'mov']
+    }
+
+    /**
+     * Get Accounts - returns the account details associated
+     * with an API key.
+     * @param {Object} obj
+     * @param {string} obj.location
+     * @param {boolean} obj.generateAccessTokens
+     * @param {boolean} obj.allowEdit
+     * @returns {Promise<[Object]>} A promise containing an array of objects
+     */
+    getAccounts({

```

Comment:
Is this specific to the video indexer api? If not, can we pull this out into a separate file?